Library/src/Multiplayer/MultiplayerConnection.cpp

```diff
@@ -450,21 +455,29 @@ void MultiplayerConnection::StartEventMessageListening()
 				SequenceChangedCallback(SequenceDeserialiser.GetEventParams());
 			}
 
-			auto EntitySystem = csp::systems::SystemsManager::Get().GetSpaceEntitySystem();
-
-			if (EntitySystem->SequenceHierarchyChangedCallback)
-			{
-				// Only fire the SequenceHierarchy callback if its an EntityHierarchy sequence
-				csp::common::String Key			 = SequenceDeserialiser.GetEventParams().Key;
-				csp::common::String SequenceType = GetSequenceKeyIndex(Key, 0);
-
-				if (SequenceType == "EntityHierarchy")
-				{
-					SequenceHierarchyChangedEventDeserialiser Deserialiser;
-					Deserialiser.Parse(EventValues);
-					EntitySystem->SequenceHierarchyChangedCallback(Deserialiser.GetEventParams());
-				}
-			}
+            // There are a variety of sequence types.
+            // Other CSP callbacks may also need to fire if the sequence change relates to a particular sequence type.
+			const csp::common::String Key = SequenceDeserialiser.GetEventParams().Key;
+            const csp::common::String SequenceType = GetSequenceKeyIndex(Key, 0);
+            if (SequenceType == "EntityHierarchy")
+            {
+				auto EntitySystem = csp::systems::SystemsManager::Get().GetSpaceEntitySystem();
+                if(EntitySystem->SequenceHierarchyChangedCallback)
```

Comment:
picky but inconsistent use of space after if statement (Michael picked me up on something like this so blame him)

Reply:
You got it!

Reply:
Fixed in latest.

Reply:
Still seeing a few 'if's without spaces. Not a big deal fi we aren't that concerned.

Reply:
Oh, that. Yes I'm not personally concerned and it's not actually called out in the coding standards. If you're happy, I'm happy.